pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+filterwarnings =
+    ignore:.*does not support directional derivatives.*:UserWarning
```

Comment:
Config file for pytest, ingore all warnings matching the regex "*does not support directional derivatives.*" and of type UserWarning